compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -423,6 +423,14 @@ fn get_pgo_sample_use_path(config: &ModuleConfig) -> Option<CString> {
         .map(|path_buf| CString::new(path_buf.to_string_lossy().as_bytes()).unwrap())
 }
 
+fn get_instr_profile_output_path(config: &ModuleConfig) -> Option<CString> {
+    if config.instrument_coverage {
+        Some(CString::new(format!("{}", PathBuf::from("default_%m_%p.profraw").display())).unwrap())
```

Comment:
I think we can just do this right? 

```suggestion
        Some(CString::new("default_%m_%p.profraw").unwrap())
```

Reply:
Updated.